modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -203,47 +203,73 @@ Each provider has its own configuration map that can be passed to the `genai.vec
 | model
 | STRING
 | The name of the model you want to invoke. +
-Supported values: `"textembedding-gecko@001"`
+ +
+Supported values: +
+ +
+
+label:new[Introduced in 5.17]:
+
+* `"textembedding-gecko@001"`
+
+label:new[Introduced in 5.19]:
+
+* `"textembedding-gecko@002"`
+* `"textembedding-gecko@003"`
+* `"textembedding-gecko-multilingual@001"`
+
 | `"textembedding-gecko@001"`
```

Comment:
Repeated above?

Reply:
This one should be for the "default" column, whereas the one above is for "supported values".